Tools/scripts/deepfreeze.py

```diff
@@ -278,6 +279,7 @@ def generate_code(self, name: str, code: types.CodeType) -> str:
             self.write(f".co_name = {co_name},")
             self.write(f".co_qualname = {co_qualname},")
             self.write(f".co_linetable = {co_linetable},")
+            self.write("._co_linearray = NULL,")
```

Comment:
Sorry, unrelated question, but does this mean I need to add the field from my `_co_code` change here too?

Reply:
Technically, no. C will initialize any fields not explicitly mentioned to zero.
I think it is good practice to be explicit, though.